packages/mux-player-react/REFERENCE.md

```diff
@@ -1,59 +1,60 @@
+| `hotkeys`                  | `string` (space separated enum list)                                | Similar to `className`, a space-separated string blocklist of keyboard shortcuts (hotkeys) you want to disable. Based on [Media Chrome's implementation](https://media-chrome-docs.vercel.app/en/keyboard-shortcuts)                                                                                                                                                                             | `''`                                                              |
+| `nohotkeys`                | `boolean`                                                           | Toggles keyboard shortcut (hot keys) support when focus in inside the player                                                                                                                                                                                                                                                                                                                     | `false`                                                           |
+| `thumbnailTime`            | `number` (seconds)                                                  | [Time offset](https://docs.mux.com/guides/video/get-images-from-a-video#thumbnail-query-string-parameters) for the default `poster` image based on your `playback-id`. If no `thumbnail-time` is specified, `start-time` will be used by default. NOTE: This feature currently cannot be used with `token.thumbnail`.                                                                            | `0`                                                               |
+| `title` (**deprecated**)   | `string`                                                            | Title text to show for your content in the Mux Player UI. (**deprecated**: Use `videoTitle` instead. In most browsers the `title` attribute causes a tooltip on hover which is undesired hence in the next major Mux player version `title` will not cause any Mux player UI changes anymore. Only the native browser behavior will be kept.)                                                    | `""`                                                              |
+| `videoTitle`               | `string`                                                            | Title text to show for your content in the Mux Player UI.                                                                                                                                                                                                                                                                                                                                        | `""`                                                              |
```

Comment:
issue(blocking): Update default to describe relationship to `title`. See other "inferred" callouts for examples.